config/versions.cfg

```diff
@@ -1,31 +1,73 @@
 [versions]
 collective.recipe.supervisor = 1.0.0
+haufe.requestmonitoring = 0.6.0
+path.py = 12.5.0
+sentry-sdk = 1.45.0
+slc.zopescript = 2.0.0a1
+superlance = 2.0.0
+zope.app.publication = 4.5
+zope.session = 4.5
+
+# Required by:
+# recensio.plone==2.0.0.dev0
 collective.vdexvocabulary = 0.3
+
+# Required by:
+# recensio.plone==2.0.0.dev0
 eea.facetednavigation = 16.2
+
+# Required by:
+# recensio.plone==2.0.0.dev0
 ftw.upgrade = 3.3.1
```

Comment:
Please use the latest versions you find here: https://pypi.syslab.com/packages/ otherwise the upgrade script will not work.